07-routing-gossip.md

```diff
@@ -316,12 +317,6 @@ The receiving node:
 any future fields appended to the end):
     - SHOULD fail the connection.
     - MUST NOT process the message further.
-  - if `features` field contains _unknown even bits_:
```

Comment:
if i understand correctly, this will removes the ability to partition the gossip network as a result of nodes not understanding an even feature bit? do we still want to retain the ability to do so via unknown TLV types?

Reply:
Hmm, that's an excellent question!  I... don't know.  Allowing "invalid" TLVs to propagate seems like a burden on implementations, but maybe it's a feature somehow?

Reply:
In due time we may discover, I see arguments for both sides 🤷‍♂️